api/pkg/apps/webapp/casetypes.go

```diff
@@ -187,3 +197,16 @@ func (h *Server) NewCaseType(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 }
+
+func parseTemplate(str string) (cms.CaseTemplate, error) {
+	var template []cms.CaseTemplateFormElement
+
+	err := json.Unmarshal([]byte(str), &template)
+	if err != nil {
+		return cms.CaseTemplate{}, err
+	}
+
+	return cms.CaseTemplate{
+		FormElements: template,
+	}, nil
+}
```

Comment:
Instead of this, implement a method on the CaseType
```
func (c *CaseType) UnmarshalFormData(values url.Values) error {
   caseType.Name = values.Get("name")
   caseType.PartyTypeID = values.Get("partyTypeId")
   caseType.TeamID = values.Get("teamId")
   // also implement for caseType.Template
}
```